server/test/clap.spec.js

```diff
@@ -13,7 +13,7 @@ describe('Articles routes', () => {
       .post('/api/v1/auth/login')
       .send({
         email: 'mofe@okoro.com',
-        password: 'soldier'
+        password: 'soldier123'
```

Comment:
Any particular reasons why you changed the passwords?

Reply:
@Chiazokam changed password because i added validation for alphanumeric values, so modified all seeded passwords

Reply:
Alright